### PR TITLE
serdar-korur cRew Shiny App

### PR DESCRIPTION
**Pull request by korur**
Thanks for contributing your submission to the e-Rum2020 CovidR contest.

**For CovidR contributors:**

- [x] The title of this PR reflects your contribution
- [x] The pull request only includes one new `.yml` file under `contest/contributions`
- [x] Make sure "Allow edits from maintainers" is checked
- [x] Send an email to covidrcontest@erum.io (for further direct communication down the road)
     - Subject: "Submission to CovidR: PR #00", where #00 is the number of this Pull Request
     - Content: A full link to this Pull Request
     - Optionally include your social media profiles (Twitter, LinkedIn, Facebook): we will use them to `@mention` you while sharing the contribution through our channels.

---

For CovidR maintainers:

- [x] Label the PR as "submission"
- [x] Check the validity of the submission
- [x] Assess completeness of metadata and possibly adjust the `.yml` file-name
- [x] Check the integration of the new page in the website gallery (via Actions' artifact or locally)
- [x] Make sure the email to covidrcontest@erum.io was received
- [x] Add a field with the current `date` and approx. merge time (e.g. `2020-04-01-16:30` for April 1st, 4:30 PM)
- [x] Squash-merge the pull request (commit headline will be OK, consider cleaning-up the details)
- [x] The initial comment should be automatically created, otherwise comment manually with the raw content of [vote-comment.md](../tree/master/contest/_tools/vote-comment.md) and an initial like
- [x] Provide here the link to the contribution as published to the gallery: https://milano-r.github.io/erum2020-covidr-contest/serdar-korur.html
- [x] Instruct contributors on how to include their badge from the gallery page, share on social media, and get a [free ticket](https://www.eventbrite.it/e/covidr-erum-2020-pre-conference-event-tickets-104133026686).
- [x] Consolidate e-mailed information.

### CODE REVIEW

**Comment by riccardoporreca** [comment]
@korur, I checked you contribution at https://tools.dataatomic.com/shiny/cRew/, but I cannot see much in any of the section... is it due to the fact that the app was just launched and no data have been collected yet? Or is it meant to only display the users' data once they are entered? Note I am not logging in.

**Comment by korur** [comment]
Hi Ricardo,

Thanks for contacting, indeed panels become visible after logging in.

Here is a video i made today, showing how it looks after logging in.
https://www.youtube.com/watch?v=Fobss1YFRFI&feature=youtu.be

It needs authorization to keep track of user history and give personalized
results (e.g. risk analysis around your location etc.) and to ensure
preventing unauthorized writes on the database.
(This is the functionality
https://firebase.john-coene.com/reference/reqsignin )


I forgot to mention in my first email my social links are
twitter: @dataatomic
linkedin: https://www.linkedin.com/in/serdar-korur/

  Let me know if you have any other questions.

Best,
Serdar

**Comment by riccardoporreca** [comment]
@korur, thanks for the explanation.

The fact that the app just says "Login to enable personalized features" is perhaps a bit misleading, since in fact (as stated in the [README](https://github.com/korur/cRew/blob/master#about)):
> The app uses Google firebase authentication. This ensures that only authenticated users are able to interact with the app. 

I would suggest mentioning something along those lines in the abstract, where you may also want to include a link to https://github.com/korur/cRew#readme for more info (where e.g. the YT video is also mentioned). Markdown syntax is supported in the `abstract` YAML field.

You could also think of being more explicit in the app itself ("Login to enable personalized features"), so that users who just see the app w/o reading the abstract or the README are not surprised (as I was :smiley:), and this change can also happen after the submission is finalized.

Looking forward to your abstract update, We will then be happy to integrate your submission!

**Comment by riccardoporreca** [comment]
Thanks @korur for the prompt action, I will soon integrate your contribution.

File: contest/contributions/serdar-korur.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  Covid-19 pandemic caught us off-guard. To avoid this happening again we probably need tools
+  that enable us to react quickly when a new disease emerges and set measures to minimize its spread. 
+  In an attempt in this direction, we developed cRew. 
+  
+  coRanavirus early-warning (cRew) tracks Covid-19 / flu like disease symptoms in real-time. 
+  The goal is to map in real-time healthy and symptomatic people. As users enter data about 
+  their health status, the app monitors temporal and spatial changes and estimates sudden 
+  increases or decreases on local risks. 
+  The app is built with shinyMobile & firebase, echarts4r and fireData packages.
+menu_entry: cRew
```
Comment (riccardoporreca) [inline contest/contributions/serdar-korur.yml]: Improved menu entry
```suggestion
menu_entry: cRew Symptom Tracker
```

**Review by riccardoporreca — APPROVED** [review]

**Comment by riccardoporreca** [comment]
@korur, your submission is now accepted and visible at https://milano-r.github.io/erum2020-covidr-contest/serdar-korur.html :tada: 
You can click on the **eRum2020::Covidr badge** in the page to obtain the badge code you can embed in your repository.

Furthermore, feel free to spread your contribution on social media with the link above, using hashtags `#erum2020` `#covidr`, and invite the community to like the contribution using the corresponding button.

Last but not least: You can get your **free ticket for the pre-conference event** at https://www.eventbrite.it/e/covidr-erum-2020-pre-conference-event-tickets-104133026686

**Pull request merged**